vm/test.c

```diff
@@ -53,20 +55,27 @@ int main(int argc, char **argv)
         { .name = "register-offset", .val = 'r', .has_arg=1 },
         { .name = "unload", .val = 'U' }, /* for unit test only */
         { .name = "reload", .val = 'R' }, /* for unit test only */
+        { .name = "defensive", .val = 'd' },
         { }
     };
 
     const char *mem_filename = NULL;
     bool jit = false;
     bool unload = false;
     bool reload = false;
+    bool mitigations = false;
 
     int opt;
-    while ((opt = getopt_long(argc, argv, "hm:jr:UR", longopts, NULL)) != -1) {
+    while ((opt = getopt_long(argc, argv, "hm:djr:UR", longopts, NULL)) != -1) {
         switch (opt) {
         case 'm':
             mem_filename = optarg;
             break;
+        case 'd':
+            // Seed random number generator if mitigations are enabled.
+            srand(time(NULL));
```

Comment:
srand seeds the RNG for rand()
srandom seeds the RNG for random()
```suggestion
            srandom(time(NULL));
```